pkg/windows/windows.go

```diff
@@ -256,23 +221,68 @@ type windows struct {
 	log      logr.Logger
 	// defaultShellPowerShell indicates if the default SSH shell is PowerShell
 	defaultShellPowerShell bool
+	// filesToTransfer is the list of files needed for the windows VM
+	filesToTransfer map[*payload.FileInfo]string
+}
+
+func getFilesToTransfer(platform configv1.PlatformType) (map[*payload.FileInfo]string, error) {
```

Comment:
By moving this function within the file you have made the diff a lot bigger. Please be careful when doing this as it makes the review more difficult. It should be moved back to where it was, line 169 or so.